README.md

```diff
@@ -109,6 +109,31 @@ in {
 }
 ```
 
+### Integrating with HomeManager
+
+Integrating with the home-manager can be done by adding your modules to the `imports` attribute.
+You can then configure your services like usual.
+
+```nix
+let
+  nur-no-pkgs = import (builtins.fetchTarball "https://github.com/nix-community/NUR/archive/master.tar.gz") {};
+in
+{
+  imports = lib.attrValues nur-no-pkgs.repos.moredhel.home-manager.modules;
```

Comment:
`home-manager.modules` feels a bit verbose, but I also cannot think of a shorter version.

Reply:
I considered `hm` initially, but decided on explicitness for the moment.

personally I think it would make things nicer (also, autocompletion works better without `-`). And I can add a comment to the template repo that it refers to home-manager

Reply:
Perhaps we could agree on `home-modules`.

Reply:
You can see here what I now use, I can update this PR to reflect it.

I also prefer to lean on the side of correctness with a shorthand for those that want it.

https://github.com/moredhel/nur-packages/blob/master/default.nix#L17

Reply:
Ooh, I get how to use it @moredhel 
I am just thinking it'd be nice to agree on a convention on where to put home-manager modules.
I think `home-modules` would make sense, though, alternatively `home-manager.modules` is good. I am wondering what I should use when I get to moving my [modules](https://github.com/Jomik/dotfiles/tree/1dfa4b2c3cbd48328a697412f90e14a8bf5d16b3/.config/nixpkgs/modules) into my nur repo.

Reply:
Perhaps `nur-no-pkgs.repos.moredhel.hm-modules` or `nur-no-pkgs.repos.moredhel.hmModules`?

Reply:
FWIW, I just added my HM module for [generating Emacs init files](https://gitlab.com/rycee/nur-expressions/blob/master/hm-modules/emacs-init.nix) to `nur-no-pkgs.repos.rycee.hmModules.emacs-init` but would be happy to move it elsewhere if there is a general preference for a different attibute path.

Reply:
I'm happy to move it to hmModules 👍

Reply:
I also added a nix-darwin module yesterday, under `ndModules` 🙂

I'm not super happy about these names, though. My preference otherwise would be `modules.nixos`, `modules.home-manager`, and `modules.nix-darwin` but this conflicts with the existing `modules` attribute semantics.

Reply:
I think you bring up a good point as I am wanting to pull out some of my crostini-specific conf into something like `crosModules`.

I think for the moment this PR is good to go though, and we can discuss what the best course of action is (if any) regarding namespacing NixOs modules.